src/main/bash/sdkman-env.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+#
+#   Copyright 2017 Marco Vermeulen
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+function __sdk_env {
+  if [[ ! -f .sdkmanrc ]]; then
+    __sdkman_echo_red '.sdkmanrc not found.'
+
+    return 1
+  fi
+
+  local candidate version
+
+  while IFS='=' read -r candidate version; do
+    __sdk_use "$candidate" "$version"
+  done < .sdkmanrc
```

Comment:
Can we extract `.sdkmanrc` to a constant? It referenced twice so far.

Reply:
I'm more of a rule of three kind a guy but I guess it doesn't hurt.

Reply:
Me three :smile:, but in this case the string literal could be error prone.